### ISSUE TITLE
Missing keys in task documentation

### ISSUE THREAD

**Issue by cbrxyz** [amazingmarvin/MarvinAPI#15]
The following keys are missing from the task documentation:
```
createdAt
timeZoneOffset
workedOnAt
updatedAt
onboard
rewardPoints
sectionId
echoedAt
rewardId
generatedAt
remind
deletedAt
restoredAt
subtaskList
timeBlockSectionSection
imported
```

Is there anyway descriptions/types can be added for these keywords in the Task documentation?

Also, can an explanation of how the goal-related keys work, like `g_in_GOAL`, `g_sec_GOAL`, etc. be added?

**Comment by cbrxyz** [comment]
All of the `remind`s are set to `None`. I found only a few instances of those though in all of my several thousand tasks. I don't see any discernible pattern for which tasks they are set on, but all of the tasks that that key is on had reminders in the past using the real reminder key values.

All of the `timeBlockSectionSection`s are set to `0`. I found a few more of those out of all my tasks, but not that many, probably around 10-15 instances. All of the tasks with this key were originally put into Marvin through the email to Marvin strategy.

**Comment by amazingmarvin** [comment]
@cbrxyz ah ok, I've fixed the `timeBlockSectionSection` bug. Thanks. Could it be that it's very old tasks that have a `remind` field?

**Comment by cbrxyz** [comment]
I don't think so... some of the tasks with the key were as recent as last December or so. All of the tasks were given a reminder at some point though.
